npm/css-src/sass/overrides/_progress.scss

```diff
@@ -0,0 +1,32 @@
+.progress-display[value] {
+
+    /* Get rid of the default appearance */
+    appearance: none;
+    border: none;
+    /* Dimensions */
+    width: 100%; height: 20px;
+    background-color: rgb(245,245,245);
+    border-radius: 10px;
+    box-shadow: 0 2px 3px rgba(0,0,0,.5) inset;
+    color: rgb(65, 105, 225);
+    position: relative;
+    margin: 0 0 1.5em;
+}
+
+.progress-display[value]::-webkit-progress-bar {
+    background-color: rgb(245,245,245);
+    border-radius: 10px;
```

Comment:
Could you create variables for things like the border-radius which are repeated in the different blocks? That will mean it only has to change in one place if we need to tweak it later.